FusionMCPBridge/.env.example

```
@@ -0,0 +1,5 @@
+# Fusion 360 Python API path
+# Replace with your actual Fusion 360 installation path
+# macOS: /Users/<username>/Library/Application Support/Autodesk/webdeploy/production/<version>/Api/Python/packages
+# Windows: C:\Users\<username>\AppData\Local\Autodesk\webdeploy\production\<version>\Api\Python\packages
+PYTHONPATH=${HOME}/Library/Application Support/Autodesk/webdeploy/production/<version>/Api/Python/packages
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)